docs/install.md

```diff
@@ -32,14 +32,17 @@ You can also be deploy the MLRun Service using local Docker for demo and test pu
 You can work with your favorite IDE (e.g. PyCharm, VSCode, Jupyter, Colab, etc.). Read how to configure your client against the deployed
 MLRun server in {ref}`install-remote`.
 
-Once you have installed and configured MLRun, follow the [Quick Start tutorial](https://docs.mlrun.org/en/latest/tutorials/01-mlrun-basics.html) and additional {ref}`Tutorials and Examples<tutorial>` to learn how to use MLRun to develop and deploy machine learning applications to production.
+Once you have installed and configured MLRun, follow the [Quick Start tutorial](./tutorials/01-mlrun-basics.ipynb) and additional {ref}`Tutorials and Examples<tutorial>` to learn how to use MLRun to develop and deploy machine learning applications to production.
 
 
 <a id="MLRun-client-backward-compatibility"></a>
 ### MLRun client backward compatibility  
 
 Starting from MLRun v1.3.0, the MLRun server is compatible with the client and images of the previous two minor MLRun releases. When you upgrade to v1.3.0, for example, you can continue to use your v1.1- and v1.2-based images, but v1.0-based images are not compatible.
 
+After you update the MLRun package client version by running `pip install mlrun==<"new-client-version">`, you must update the images to use the same client version you installed.
+For example, when running this command `pip install mlrun==1.8.0` you must update your images to use MLRun v1.8.0 by adding `mlrun==<"new-client-version">` as a function requirement. See [mlrun.set_environment](https://docs.mlrun.org/en/stable/api/mlrun.html#mlrun.set_environment).
```

Comment:
the link should point [with_requirements](https://docs.mlrun.org/en/stable/api/mlrun.runtimes/mlrun.runtimes.html#mlrun.runtimes.BaseRuntime.with_requirements)